src/components/JobPost/DatePicker.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import DatePicker, { DayValue } from 'react-modern-calendar-datepicker'
+
+const DateSelector = () => {
```

Comment:
This file should be in the components folder so that it's clear and usable in other pages too and named according to the function: DateSelector.

Reply:
Done